server/src/Assets/dto/update-asset.dto.ts

```diff
@@ -0,0 +1,4 @@
+import { PartialType } from '@nestjs/mapped-types';
+import { CreateAssetDto } from './create-asset.dto';
+
+export class UpdateAssetDto extends PartialType(CreateAssetDto) {}
```

Comment:
@jvendettuoli this follows the same pattern you used with `UpdateUserDto`
do you remember why you had a separate DTO here rather than just reuse the `CreateUserDto` (which i think we would have all of the same properties of)

Reply:
@Sunset05 give it 5 minutes of work to see if we can just reuse the createDto